app/controllers/shows_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
   end
 
   def new
-    show_season = (params[:season] || Time.this_season).to_i
+    show_season = permit_new_show.to_i
```

Comment:
function changed name from `season_new_params`

Reply:
`permit_new_show` was the old function name

Reply:
resolved, sorry